src/js/stores/search.js

```diff
@@ -294,26 +315,26 @@ define(function (require, exports, module) {
                         var idType = kind,
                             title = CATEGORIES[kind];
 
-                        var categories = [header],
+                        var itemCategories = [header],
                             id = "FILTER-" + header;
 
                         if (header !== idType) {
-                            categories = [header, idType];
+                            itemCategories = [header, idType];
                             id += "-" + idType;
                         }
                         
-                        var icon = svgUtil.getSVGClassesFromFilter(categories);
+                        var icon = this.getSVGClass(itemCategories);
                         
                         filters.push({
                             id: id,
                             title: title,
                             svgType: icon,
-                            category: categories,
+                            category: itemCategories,
                             style: { "fontStyle": "italic" },
                             haveDocument: searchInfo.haveDocument,
                             type: "filter"
                         });
-                    });
+                    }.bind(this));
```

Comment:
Hot tip: Both `Array.prototype.forEach` and `Immutable.List.prototype.forEach` (and all their collection-traversing friends) allow you to pass a final "receiver" parameter so you don't have to explicitly bind a `this` to the function.